projects/plugins/jetpack/extensions/blocks/ai-image/edit.js

```diff
@@ -14,6 +14,31 @@ import { useSelect, useDispatch } from '@wordpress/data';
 import { useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
+function getRandomItem( arr ) {
+	// get random index value
+	const randomIndex = Math.floor( Math.random() * arr.length );
+	return arr[ randomIndex ];
+}
+
+const PLACEHOLDER = getRandomItem( [
```

Comment:
Let's have this list in seperate file

Reply:
Yeah different file would be easier to update over time

Reply:
Done